src/cdk/listbox/listbox.ts

```diff
@@ -783,7 +783,9 @@ export class CdkListbox<T = unknown> implements AfterContentInit, OnDestroy, Con
    * @param event The focusout event
    */
   protected _handleFocusOut(event: FocusEvent) {
-    const otherElement = event.relatedTarget as Element;
+    // Some browsers (e.g. Chrome and Firefox) trigger the focusout event when the user switches tabs and returns.
+    // In this case, event.relatedTarget is null, so we use document.activeElement as a fallback.
+    const otherElement = (event.relatedTarget ?? document.activeElement) as Element;
```

Comment:
I think that this is risky because we don't know if the `focusout` happened because of a tab switch or because the user actually left the listbox. Couldn't we restore the `activeTabIndex` when focus comes back to the current tab?

Reply:
mmm...I don't get risk that you are talking about. they focused out of the listbox either way and we check to see if the next focused element is either the listbox itself or a child of it...right?

I thought of maybe using `Page Visibility API`, but something surprising happens in Firefox (115)...when switching fast between tabs, it triggers 'focusout' but not the 'visibiltychange'. :/

Reply:
My understanding is that during a "regular" `focusout` event (one within the same tab) the `document.activeElement` might still be the element that is losing focus, rather than the element that focus went to. I'm concerned that if the `focusout` happens within the listbox and the browser decides not to set a `relatedTarget` for some reason, we wouldn't have a way of distinguishing it from the users switching tabs.

As for detecting when the user has switched to a different tag: I think that the `blur` handler on `window` should be enough.

Reply:
Sorry for the delay. Could you please take a look?
Appreciate your help. ;)